From 730876d067f24380ccec1bdd1f179a664f11aa2f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?P=C3=A1draig=20Brady?= <P@draigBrady.com>
Date: Thu, 2 Apr 2020 16:10:05 +0100
Subject: [PATCH 04/13] maint: clean up recently added test

* tests/misc/uniq-collate.sh: Remove logic that
was already refactored into gen_input().
---
 tests/misc/uniq-collate.sh | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/tests/misc/uniq-collate.sh b/tests/misc/uniq-collate.sh
index 974befce2..997679def 100755
--- a/tests/misc/uniq-collate.sh
+++ b/tests/misc/uniq-collate.sh
@@ -28,17 +28,17 @@ gen_input()
 
 # strcoll() used to return 0 comparing the following strings
 # which was fixed somewhere between glibc-2.22 and glibc-2.30
-gen_input '%s\n' 'ⁿᵘˡˡ' 'ܥܝܪܐܩ' > in || framework_failure_
+gen_input '%s\n' 'ⁿᵘˡˡ' 'ܥܝܪܐܩ'
 test $(LC_ALL=$LOCALE_FR_UTF8 uniq < in | wc -l) = 2 || fail=1
 
 # normalization in strcoll is inconsistent across platforms.
 # glibc based systems at least do _not_ normalize in strcoll,
 # while cygwin systems for example may do so.
 # á composed and decomposed, are generally not compared equal
-gen_input '\u00E1\na\u0301\n' > in || framework_failure_
+gen_input '\u00E1\na\u0301\n'
 test $(LC_ALL=$LOCALE_FR_UTF8 uniq < in | wc -l) = 2 || fail=1
 # Similarly with the following equivalent hangul characters
-gen_input '\uAC01\n\u1100\u1161\u11A8\n' > in || framework_failure_
+gen_input '\uAC01\n\u1100\u1161\u11A8\n'
 test $(LC_ALL=ko_KR.utf8 uniq < in | wc -l) = 2 || fail=1
 
 # Note if running in the wrong locale,
@@ -46,10 +46,10 @@ test $(LC_ALL=ko_KR.utf8 uniq < in | wc -l) = 2 || fail=1
 # I.e., cjk and hangul will now work even if
 # uniq is running in the wrong locale
 # hangul (ko_KR.utf8)
-gen_input '\uAC00\n\uAC01\n' > in || framework_failure_
+gen_input '\uAC00\n\uAC01\n'
 test $(LC_ALL=en_US.utf8 uniq < in | wc -l) = 2 || fail=1
 # CJK (zh_CN.utf8)
-gen_input '\u3400\n\u3401\n' > in || framework_failure_
+gen_input '\u3400\n\u3401\n'
 test $(LC_ALL=en_US.utf8 uniq < in | wc -l) = 2 || fail=1
 
 # Note strcoll() ignores certain characters,
@@ -57,7 +57,7 @@ test $(LC_ALL=en_US.utf8 uniq < in | wc -l) = 2 || fail=1
 # I.e., the following on glibc-2.30 at least,
 # as expected, does not print a single item,
 # but testing here for illustration
-gen_input ',a\n.a\n' > in || framework_failure_
+gen_input ',a\n.a\n'
 test $(LC_ALL=$LOCALE_FR_UTF8 uniq < in | wc -l) = 2 || fail=1
 
 Exit $fail
-- 
2.27.0

